pkg/scheduler/scheduler.go

```diff
@@ -466,7 +466,9 @@ func (sched *Scheduler) scheduleOne() {
 					" No preemption is performed.")
 			} else {
 				preemptionStartTime := time.Now()
-				sched.preempt(pod, fitError)
+				if err := sched.preempt(pod, fitError); err != nil {
+					klog.Warningf("%v/%v preempting lower priority pods fail,%v", pod.Namespace, pod.Name, err)
```

Comment:
please change to `... pods fail: %v`

Reply:
what would be the report rate of this warning?
does it make sense to only show it in verbose mode - e.g. `V(2)`?

Reply:
Agreed, increasing log level will ensure we are not aggressively filling logs.